client/utils/currency/index.js

```diff
@@ -65,10 +65,11 @@ export const getCurrency = ( currencyCode, baseCurrencyCode = null ) => {
 				}
 			}
 		}
-		return new Currency( currency );
+		return Currency( currency );
```

Comment:
It used to be a class (with a constructor) in older package versions, but it's just a function since then.